packages/frontend-shared/src/gql-components/ChooseExternalEditor.cy.tsx

```diff
@@ -39,7 +39,7 @@ describe('ChooseExternalEditor', { viewportHeight: 400, viewportWidth: 300 }, ()
 
     cy.get('[data-cy="custom-editor"]').should('not.exist')
 
-    cy.get('@chooseEditor').click()
+    cy.contains('Vim').click()
```

Comment:
Are this and `ChooseExternalEditorModal.cy.tsx`'s changes are to avoid flake unrelated to this PR?

Reply:
No, it's actually a required change. 

```
    cy.contains(defaultMessages.settingsPage.editor.noEditorSelectedPlaceholder)
    .should('be.visible')
    .as('chooseEditor')
```

This previously selected a *specific element* as the alias - since .should() was not a query, you ended up with the alias `chooseEditor: [<input>]`. 

Since .should() is now a query, you instead ended up with the alias `chooseEditor: [undefined, .contains('Choose Editor'), .should('be.visible')]`. Once we've selected an editor, the alias doesn't match anything on the page; there's no element that contains "choose editor" any more.

This could instead be fixed by moving to a test selector, something like

```
    cy.get('[cy-data'="choose-editor"]).as('chooseEditor')

    cy.get('@chooseEditor').contains('Choose Editor').should('be.visible')
```

but I found it more convenient to make this change change instead (just re-selecting an element that now contains 'Vim') rather than relying on an alias.